spec/spec_helper.rb

```diff
@@ -1,17 +1,18 @@
 # frozen_string_literal: true
 
-# Make sure specs run with the definitions from test.rb
-ENV['ROBOT_ENVIRONMENT'] = 'test'
-
-require 'webmock/rspec'
 require 'simplecov'
-require 'byebug'
 
 SimpleCov.start do
   add_filter 'spec'
   add_filter 'config'
 end
 
-require File.join(File.dirname(__FILE__), '..', 'config/boot')
+ENV['ROBOT_ENVIRONMENT'] = 'test'
```

Comment:
do we still need this, if we got rid of the config/environments?

Reply:
Yes, it is required to set up `config/settings`.